trieste/models/gpflow/__init__.py

```diff
@@ -20,4 +20,9 @@
 from .config import GPflowModelConfig
 from .interface import GPflowPredictor
 from .models import GaussianProcessRegression, SparseVariational, VariationalGaussianProcess
+# from .optimizer import (
```

Comment:
this is a current solution for the issue with importing `create_optimizer` and `create_loss_function` with the singledispatch decorator from `models/optimizers.py`, we don't want to export these internal functions, but I didn't manage to find an alternative solution